index.html

```diff
@@ -9,19 +9,28 @@
     <body>
         <div class="top">
         <header class="header">
-            <h2 class="logo"><a class="logo" href="#home">PETER</a></h2>
+            <h2 class="logo"><a class="logo" href="#top">PETER</a></h2>
     <nav class="navbar">
             <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
                 <path fill-rule="evenodd" clip-rule="evenodd"
                     d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
-                    fill="#acb7c3" />
-            </svg>              
+                    fill="#acb7c3" class="mobile_menu"/>
+            </svg>  
+            
+            <div id="mobile">
+                <ul id="menu_top">
+                <div id="close-button">✕</div>
+                  <li><a href="#home">Portfolio</a></li>
+                  <li><a href="#about">About</a></li>
+                  <li><a href="#contact">Contact</a></li>
+                </ul>
+              </div>            
         
         <ul class="navigation">
-            <li class="portfolio"><a href="#home">portfolio</a></li>
+            <li class="portfolio"><a href="#top">portfolio</a></li>
             <li class="abt"><a href="#about">About</a></li>
-            <li class="Contact"><a href="#contact">Contact</a></li>
-            <i class='fas'>&#xf199;</i>
+            <li class="Contact"><a href="#connect">Contact</a></li>
+            <i class='fas'><a  href="mailto:peteronyam@gmail.com">&#xf199;</a></i>
```

Comment:
- [OPTIONAL] I am going to make this an optional change because I missed pointing it out in the previous review, :1st_place_medal:  Following the best `HTML` practice for writing dry code, you are not meant to have two different menus in your code base, You are meant to have just one and then dynamically adapt it to fit both mobile and desktop. Please you need to fix this as your project is meant to follow the best practices. :smile: :smile: